plugins.d/tag.global.plugin

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+function tag.help {
+   # Select either of the two or both, depending on the functionality provided by the plugin
+   echo "  oc-cluster show-tags <ImageStreamName>"
+   echo "  oc-cluster tag-latest <ImageStreamName> <tag-name> - tags 'latest' ImageStream to <tag-name>"
+   echo "  oc-cluster tag-to-target <ImageStreamName> <source-tag> <target-project> <target-tag>"
+   echo "      Assumes that <source-tag> is something like 'RELEASE_1_1' and <target-tag> is 'latest'"
+   echo "      It first 'copies' the 'RELEASE_1_1' into <target-project> and call it 'latest'"
+   echo "      and then tags <target-project>:latest to 'RELEASE_1_1'"
+}
+
+#
+# Args:
+#  $1 ImageStreamName - the name of the application to tag
+#  $2 source-tag - the source tag to use
+#  $3 target-project - the name of the target project
+#  $4 target-tag - the name of the target tag in target-project
+#
+function tag-to-target {
+	[ $# -lt 4 ] && echo "ImageStream Name is required." && exit 1
+	local IMAGE_STREAM=$1
+	local SOURCE_TAG=$2
+	local TARGET_PROJECT=$3
+	local TARGET_TAG=$4
+	local SOURCE_PROJECT=$(oc project -q)
+	
+	if [[ $SOURCE_PROJECT == $TARGET_PROJECT ]]; then
+		echo "$SOURCE_PROJECT equals $TARGET_PROJECT. Use tag-latest instead."
+		exit 1
+	fi
+	
+	echo "Tagging '$SOURCE_TAG' image stream '$IMAGE_STREAM' from '$SOURCE_PROJECT' to target project '$TARGET_PROJECT' with tag '$TARGET_TAG'..."
+		
+	oc tag $SOURCE_PROJECT/$IMAGE_STREAM:$SOURCE_TAG $TARGET_PROJECT/$IMAGE_STREAM:$TARGET_TAG
+	oc tag $TARGET_PROJECT/$IMAGE_STREAM:$TARGET_TAG $TARGET_PROJECT/$IMAGE_STREAM:$SOURCE_TAG
+	
+	oc project $TARGET_PROJECT
```

Comment:
Make this project agnostic, so that you can tag from any project to any project without you requiring to get into the project
oc label is $IMAGE_STREAM APP_VERSION=$TARGET_TAG --overwrite -n $TARGET_PROJECT --as=system:admin